src/Components/App/App.js

```diff
@@ -40,9 +68,18 @@ const App = () => {
           </Route>
           <Route exact path="/recipes" render={ () => {
             return <DisplayArea 
+              favorites={favorites}
               recipes={recipes}
+              handleFavorites={handleFavorites}
+            />
+          }} />
+          <Route exact path="/favorites" render={ () => {
```

Comment:
Pretty clear and straight forward on what's is going on here (especially with efforts to change a path name), but it seems to be exactly the same code from above. Maybe there is a way to simplify the rendering of the same component based on favorites or recipes to avoid redundancy and confusion?

Reply:
Are you saying that instead of triggering it off the path, I should conditionally render it off of something like a state in app of viewFavorites set to true or false?